test/environment/createvirtualenvironment.spec.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-unresolved
+import createVirtualEnvironment from '@locker/near-membrane-dom';
+
+describe('createVirtualEnvironment', () => {
+    it('no options', () => {
+        const evalScript = createVirtualEnvironment();
+        expect(() => evalScript('')).not.toThrow();
+    });
+    it('empty options', () => {
+        const evalScript = createVirtualEnvironment({});
+        expect(() => evalScript('')).not.toThrow();
+    });
+    it('keepAlive: true', () => {
+        const evalScript = createVirtualEnvironment({ keepAlive: true });
+        expect(() => evalScript('')).not.toThrow();
```

Comment:
☝️ You can check the `window.frames` length to see if it's larger than it was before creating the environment to see if the keepAlive keeps the iframe attached.

Reply:
Oh, good idea!